### PR TITLE
validate the length of string parameters

### PR DESCRIPTION
**Pull request by Yidingluo1**
Secretname: 1 to 64 characters
Secretdata: 1 to 6144 characters
Description: ：0 to 4096 characters

test: passed the python test

Signed-off-by: yidingluo1 <luo.yd@neusoft.com>

### CODE REVIEW

File: ehsm_kms_service/secret_manager_apis.js [hunk 1]
```diff
@@ -129,16 +133,16 @@ const createSecret = async (res, appid, payload, DB) => {
         const createTime = new Date().getTime()
         let keyBlob = ''
         let nextRotationDate = ''
-        if (!checkStringParam(secretData, true)) {
-            res.send(_result(400, 'secretData cannot be empty and must be string'))
+        if (!checkStringParam(secretData, true, 6 * 1024)) {
```
Comment (yang8621) [inline ehsm_kms_service/secret_manager_apis.js:136]: Can define as a macro or const?

Comment (Yidingluo1) [reply ehsm_kms_service/secret_manager_apis.js:136]: Length will be defined as a const

**Comment by syan10** [comment]
new commit title
"
validate the length of string parameters


Secretname: 1 to 64 characters
Secretdata: 1 to 6144 characters
Description: ：1 to 4096 characters

test: passed the python test
"

**Comment by yang8621** [comment]
> new commit title " validate the length of string parameters
> 
> Secretname: 1 to 64 characters Secretdata: 1 to 6144 characters Description: ：1 to 4096 characters
> 
> test: passed the python test "

Secretname: 1 to 64 characters
Secretdata: 1 to 6144 characters
Description: ：1 to 4096 characters

Are 0 length all handled for the three items?

**Review by syan10 — APPROVED** [review]

**Comment by Yidingluo1** [comment]
> > new commit title " validate the length of string parameters
> > Secretname: 1 to 64 characters Secretdata: 1 to 6144 characters Description: ：1 to 4096 characters
> > test: passed the python test "
> 
> Secretname: 1 to 64 characters Secretdata: 1 to 6144 characters Description: ：1 to 4096 characters
> 
> Are 0 length all handled for the three items?

0 length has been processed

**Review by HawieLong — APPROVED** [review]

**Review by syan10 — APPROVED** [review]

**Pull request closed without merging**